NFTs creation, mint and transfer

Devilla opened the pull request:
# Description
NFTs creation, mint, and transfer using Avalanche wallet.

## This PR created for* - 
- [x] DataHub Official Docs (Network Documentation, API docs etc.) changes/updates/fixes
- [x] Figment Learn Docs (Pathway, Tutorials, New Tutorials etc.) changes/updates/fixes

# Checklist*:
- [x] Create, Mint, and Transfer NFTs 🎨 on Avalanche using Avalanche Wallet. 👛 
- [x] I updated the `SUMMARY.md` file (FYI, It's needed only when you're adding/creating a new file like submitting a new tutorial, pathway section)
- [x] I checked and read the contribution guide before making this PR [**HERE**](https://docs.google.com/document/d/13LWLrWzZ34M0ldWGeDANcWxw9nEWk3AX3VwXRBIOs1M/edit)  
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Note - Above points marked with * are mandatory to fill :)

Devilla commented:
@yash-sharma1 Can you review it, thanks. 😊

yash-sharma1 commented on network-documentation/avalanche/tutorials/create-mint-transfer-nft.md:
```diff
@@ -0,0 +1,160 @@
+---
+
+# Create, Mint and Transfer NFTs using Avalanche wallet
+
```
Before the Introduction please add the "About The Author" section and put your bio in it along with your social media handle (if any)

Devilla replied:
Okay, I'll update that. Thanks.

Devilla replied:
@yash-sharma1 I have updated the files as per the requirements, please review and merge.

yash-sharma1 replied:
Thanks for the update @devilla We'll review the changes and will update you accordingly. Appreciate your work 💯

yash-sharma1 reviewed (changes requested):
all looks good to me but I think you didn't edit the same in `SUMMARY.md` file. 
Please check this guide section 7 - Contribution and publishing process to make sure to publish it carefully and to avoid any conflicts - [Check Here](https://docs.google.com/document/d/13LWLrWzZ34M0ldWGeDANcWxw9nEWk3AX3VwXRBIOs1M/edit?usp=sharing)

Devilla commented:
Done.

yash-sharma1 reviewed (commented):
@Devilla Here in your new commit for this file - `network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md` you update the existing tutorial which is from Avalanche Official Docs and I didn't get what was the reason, as this specific tutorial is from official Avalanche Docs. Can you please explain it to us before we take it forward for review and merging it :)

Devilla commented:
> @Devilla Here in your new commit for this file - `network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md` you update the existing tutorial which is from Avalanche Official Docs and I didn't get what was the reason, as this specific tutorial is from official Avalanche Docs. Can you please explain it to us before we take it forward for review and merging it :)

I have worked on some of the documentation and implementation of the [```using-truffle-with-the-avalanche-c-chain``` ](https://github.com/ava-labs/avalanche-docs/commits?author=Devilla) with avalanche and I updated their docs based on features that I find very useful or unnecessary as a developer. I have also added an update regarding that on avalanche also. [Avash dependency]( https://github.com/ava-labs/avalanche-docs/pull/141), I'll remove this commit until it's updated on avalanche official docs. Thanks.

yash-sharma1 commented on network-documentation/avalanche/tutorials/using-truffle-with-the-avalanche-c-chain.md:
```diff
@@ -6,6 +6,25 @@ description: Learn how to use Truffle with the C-Chain
 
 [**The original tutorial can be found in the AVA Labs documentation here**](https://docs.avax.network/build/tutorials/smart-contracts/using-truffle-with-the-avalanche-c-chain). 
 
```
@Devilla I think you need to remove commit on this file or revert these changes before we take it further

Devilla replied:
I've reverted the changes to the original. Thanks.

Devilla commented:
Too much conflicts here now will PR again with individual files.
